scripts/import_libcbor.py

```diff
@@ -0,0 +1,64 @@
+import os
+import tempfile
+import shutil
+import subprocess
+import re
+
+# Create a temporary directory for cloning the repository
+temp_repo_dir = tempfile.mkdtemp()
+
+try:
+    # Clone the repository into the temporary directory
+    repo_url = "https://github.com/PJK/libcbor.git"
+    clone_command = f"git clone {repo_url} {temp_repo_dir}"
```

Comment:
this script isn't using the latest release
maybe force someone to pass it in via argparse